### PR TITLE
Added delete line shortcut in macOS

### PR DESCRIPTION
**Pull request by tlegen-k**
Added shortcut to delete whole line on darwin platform.

### CODE REVIEW

**Comment by the-j0k3r** [comment]
Thanks for this contribution. A welcome addition =)

Allow me some time (maybe a while as I'm a bit busy) to make some additions to this PR and get this ready for merge.

:+1:

File: keymaps/terminus.cson [hunk 1]
```diff
@@ -21,6 +21,7 @@
 '.platform-darwin .terminus .terminal':
   'cmd-c': 'terminus:copy'
   'cmd-v': 'terminus:paste'
+  'cmd-delete': 'terminus:delete-line'
 
 '.platform-linux .terminus .terminal, .platform-win32 .terminus .terminal':
```
Comment (UziTech) [inline keymaps/terminus.cson:26]: You should add it to linux and win32 as `'ctrl-delete': 'terminus:delete-line'`:

**Review by UziTech — CHANGES_REQUESTED** [review]

File: keymaps/terminus.cson (hunk 1 above)
Comment (tlegen-k) [reply keymaps/terminus.cson:26]: I thought about that, but i believe generally ctrl-delete reserved for deleting word on windows and linux machines. Wouldn't there be a clash of shortcuts?

Comment (UziTech) [reply keymaps/terminus.cson:26]: If we are going for familiar key bindings shouldn't mac keystrokes be <kbd>Ctrl</kbd> + <kbd>u</kbd> to delete current command?

Comment (the-j0k3r) [reply keymaps/terminus.cson:26]: > Wouldn't there be a clash of shortcuts?

This is what I wanted to check and add, but if you have time to investigate and add, please =)

**Pull request closed without merging**